lib/fetching/content-services/facebook-content-service.js

```diff
@@ -26,116 +27,139 @@ FacebookContentService.prototype._doFetch = function(options, callback) {
   var self = this;
 
   if (!this._url) {
-    process.nextTick(function() { self.emit('error', new Error('Missing Facebook URL')); });
+    process.nextTick(function () {
```

Comment:
Our style guide calls for no space between `function` and `()`. See here: https://github.com/felixge/node-style-guide. Thanks.